aten/src/ATen/native/mps/OperationUtils.mm

```diff
@@ -269,6 +269,62 @@ void printTensorNDArray(const Tensor& t) {
   return cachedGraph;
 }
 
+id<MTLBuffer> scatterViewTensor(at::Tensor& dst, const at::Tensor src, id<MTLBuffer> updatesTensorBuffer) {
```

Comment:
The code seems like a duplicate of gatherViewTensor() with updatesTesnor added to it. Also, the scatterAlongAxis is in the graph always anyway (in chainView graph), no matter it's gather or scatter. So, later lets merge them into a smarter scatterGatherViewTensor() function (fewer functions-> less maintenance required).

Reply:
Can you please merge them Denis ?

Reply:
yes - will update this PR